modules/react-native/articles/store/index.js

```diff
@@ -0,0 +1,66 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit"
+import { api } from "./api"
+
+export const article_read = createAsyncThunk(
+  "articles/article_read",
+  async ({ id }) => {
+    const response = await api.article_read(id)
+    return response.data
+  }
+)
+
+export const article_list = createAsyncThunk(
+  "articles/article_list",
+  async () => {
+    const response = await api.article_list()
+    return response.data
+  }
+)
+
+const initialState = { articles: [], api: { loading: "idle", error: null } }
```

Comment:
Why are you suggesting an object rather than a list? Are you just trying to replicate the behaviour of  `CreateEntityAdapter`? Could you explain more?
I don't see any objective advantages for the change in this case, since we literally just list the articles and display individual articles

Reply:
Nvm my comment above, I didn't read you general review message. it makes sense because our backend api returned data is already normalized. 
I'll update everything, but it will also require updating the modules code for accessing the data